api/device-configure.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: device-configure
+title: Device Configuration Operators
+---
+
+> [!CAUTION]
+>  Device configuration operators are not recommended for interfacing with off-the-shelf ONIX hardware. Use [aggregate configuration operators](xref:configure) to do that instead. Aggregate operators confer the following benefits:
+> - The `address` properties of aggregate configuration operators are automatically configured whereas a device configuration operators requires manual configuration of its `address` property. This improves ease of using Open Ephys hardware.
```

Comment:
Similar comment on #26, "This improve ease of" is a little ambiguous on what it is referring to. If changes are made in either of these PRs, it should be changed in both places.

Reply:
[This comment](https://github.com/open-ephys/onix1-bonsai-docs/pull/26#discussion_r1721731208) specifically